src/v2/Apps/Show/ShowApp.tsx

```diff
@@ -16,10 +17,26 @@ interface ShowAppProps {
 export const ShowApp: React.FC<ShowAppProps> = ({ show }) => {
   if (!show) return <ErrorPage code={404} />
 
+  const ContextualLink = () => (
+    <Box my={2}>
+      <AppContainer>
+        <HorizontalPadding>
+          <ShowContextualLink show={show} />
+        </HorizontalPadding>
+      </AppContainer>
+
+      {/* This is a full screen width separator, so needs to exist outside of
+      the <AppContainer /> HOC. */}
+      <Separator as="hr" my={2} />
```

Comment:
Makes me think we should just add some support for breaking out of the container in the `Separator` component.

```tsx
<Separator fullScreen />
```
 => 
```css
left: 50%;
margin-left: -50vw;
margin-right: -50vw;
max-width: 100vw;
position: relative;
right: 50%;
width: 100vw;
```

or something

Reply:
I think we should wait to update Palette until this pattern is needed elsewhere.

I like the CSS solution though! Will give that a try and replace it. I couldn't figure out how to get that behavior without breaking out of the `<AppContainer />`.